Retake   cache hunt

RandAlThor1 opened the pull request:
Mission Pull Request Template

This Pull Request will:

Add the mission Retake made by RandAlThor to the repository.

**Mission Creation Checklist:**
- [x] Mission File
- [x] Ingame Play Area Markers
- [x] Ingame Briefings to all players
- [x] Side-based briefings
- [x] Objective Entities
- [x] Mission .conf file

**Faction(s):**
BLUFOR - Attacking
- [x] RHS_US_USMC

OPFOR - Defending
- [x] RHS_RF_MSV

**Objectives/Win Conditions:**

Cache hunt with 4 Caches

**Terrain:**

Zarichne

**Short mission description:**

Air mobile USMC company clears MSV Caches lots of urban and trench fighting.

**Slotting Ratio:**

3:2
BLU/OP

**Time limit:**

75min

**Mission Briefing Screenshot:**
![image](https://github.com/user-attachments/assets/e260b18b-6cdb-457f-87a1-6474c8ab1fbb)


**Design concept/thoughts:**

Time should transition from dark to light over the course of the mission time
As OPFOR has the equal tech advantage, I kept respawns in line with the 3:2 ratios 50/35

**Other Notes:**

OPFOR will need ambulance spawned.
PR also includes MSV Rifleman AA, USMC LAT NLAW prefabs

tanaaKa commented:
**QA Feedback**

- IMO, switch these in the faction manager to BLUFOR/OPFOR
![image](https://github.com/user-attachments/assets/7ae6ec40-a5c5-4290-9c73-09594a4ad3de)

- Rename to `Transport Heli 1` 2,3,etc...
![image](https://github.com/user-attachments/assets/b17deff2-c4f6-4088-a46f-e6ee0b7a62f5)

- BLUFOR should probably have a logistical element if OPF has one. Not sure if you thought of that or not, but maybe not necessary

- Spawn markers are 90 degrees incorrect 
![image](https://github.com/user-attachments/assets/0f770f31-6968-4005-b212-6ad270c5eb37)
![image](https://github.com/user-attachments/assets/ac98c69f-d375-4ecf-9885-134564765b41)

We should probably add this change to the prefab itself to avoid it in the future.

- 75 minutes is probably too short

- Add a ground vehicle option for BLUFOR

RandAlThor1 commented:
- lobby names changed to, BLUFOR (ATK) and OPFOR (DEF)
- Heli crews renamed
- Markers rotated
- time increased to 90 minutes
- removed 1x helo & crew, added 2x armed strykers and crews.
- removed OPFOR Logi replcaed with MAT to counter stryker addition
